automation/jenkins/aws/manageRepo.sh

```diff
@@ -215,12 +216,6 @@ function push() {
             fatal "Can't push the ${REPO_LOG_NAME} repo changes to upstream repo ${REPO_REMOTE}" && return 1
         fi
     fi
-
-    # Removing commits which have been pushed
-    if [[ -s "${commit_stage_file}" ]]; then
-        remaining_commits="$(jq --arg dir "${REPO_DIR}" 'del(.dirs[] | select(.dir == $dir))' "${commit_stage_file}")"
-        echo "${remaining_commits}" > "${commit_stage_file}"
-    fi
```

Comment:
Since the commit stage is across all repositories I think this is still required. Is it done somewhere else? 
this only wouldn't be required in automation scenarios. Running locally I think this still applies.

Reply:
The events are actually deleted as part of `pull_events_from_state()` that pulls them into a local copy. 

If you search across the executor code base, the lines highlighted are the only use of `commit_stage_file` that I could find. Happy to be corrected (would appreciate a second set of eyes) but this code looked like a left over from a refactor that would never be executed now because it would be testing an empty string.

Reply:
Ah yep, you are totally right.